package.json

```diff
@@ -65,6 +65,8 @@
     "blipp": "4.0.1",
     "bunyan": "^1.8.12",
     "chai": "^4.2.0",
+    "cli-progress": "^3.8.2",
+    "cli-table3": "^0.6.0",
```

Comment:
Can you remove ^ symbol there for the newly added dependencies

Reply:
done